Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -94,7 +94,7 @@ class NotEnoughData(ValueError):
 
 
 class OWKMeans(widget.OWWidget):
-    name = "k-Means"
+    name = "k-Means clustering"
```

Comment:
This changes the name of the widget and makes it longer. I don't think this is ok. If docs don't match, we can change the docs.

Reply:
Changing the name would make this similar to Hierachical and Louvain Clustering (though "clustering" should be capitalized).

Reply:
Hierarchical and Louvain can not be used by themselves - "clustering" is part of the name there.
k-Means is clear enough and can be used as a name by itself.
I think we should strive for short widget names, so I would not expand it in this case.
BTW, clustering is already a keyword for k-means so using it in a search lists it alongside other methods.

Reply:
Ok. Changed back widget name and changed documentation file name.